src/functions-and-arrays.js

```diff
@@ -52,14 +110,39 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+let usedWords = []
+
+function uniquifyArray(someArr) {
+  if (someArr.length < 1) {
+    return null;
+  } else {
+    for (i = 0; i < someArr.length; i++){
+      if (usedWords.indexOf(someArr[i]) === -1){
+        usedWords.push(someArr[i]);
+      }
+    }
+    return usedWords;
+  }
+}
 
 
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
+function doesWordExist(word, someArr) {
```

Comment:
Also, minor thing but the order of arguments is reversed. It doesn't matter if you know how to feed the function calls. But it won't work for the tests as they have been instructed to call howManyTimes(word, someArr). The same happens in the next function.